easybuild/easyconfigs/z/ZPAQ/ZPAQ-7.00-GCC-4.8.2.eb

```diff
@@ -0,0 +1,33 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2015 NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+##
+
+easyblock = 'MakeCp'
+
+name = 'ZPAQ'
+version = '7.00'
+
+homepage = 'http://mattmahoney.net/dc/zpaq.html'
+description = """zpaq is a free and open source (GPL v3) incremental,
+ journaling command-line archiver for Windows, Linux and Mac OS/X"""
+
+toolchain = {'name': 'GCC', 'version': '4.8.2'}
+
+sources = ['%%(namelower)s%s.zip' % ''.join(version.split('.'))]
+source_urls = ['http://mattmahoney.net/dc/']
+
+buildopts = " && pod2man zpaq.pod > zpaq.1"
```

Comment:
This is interesting:

```
$ rpm -q --whatprovides `which pod2man`
perl-5.10.1-136.el6.x86_64
```

and indeed:

```
$ module load Perl/5.16.3-goolf-1.4.10-bare
$ which pod2man
/gpfs/scratch/projects/project_gpilot/vsc40023/software/Perl/5.16.3-goolf-1.4.10-bare/bin/pod2man
```

So, adding a Perl build on top of `GCC/4.8.2` in here as a build dependency would do it!